projects/1.18/assets/easy-emerald-tools/easy_emerald/lang/zh_cn.json

```
@@ -1 +1,105 @@
+   "item.easy_emerald.ruby_axe": "红宝石斧",
+   "item.easy_emerald.ruby_shovel": "红宝石锹",
+   "item.easy_emerald.ruby_hoe": "红宝石锄",
+   "item.easy_emerald.ruby_excavator": "红宝石开掘铲",
+   "item.easy_emerald.ruby_hammer": "红宝石锤",
+   "item.easy_emerald.ruby_paxel": "红宝石多功能工具",
+   "item.easy_emerald.ruby_treeaxe": "红宝石伐木斧",
+
+   "item.easy_emerald.ruby_head": "红宝石头盔",
+   "item.easy_emerald.ruby_body": "红宝石胸甲",
+   "item.easy_emerald.ruby_leggings": "红宝石护腿",
+   "item.easy_emerald.ruby_boots": "红宝石靴子",
+
+   "item.easy_emerald.ruby_block": "红宝石块",
+   "block.easy_emerald.ruby_block": "红宝石块",
+
+   "item.easy_emerald.obsidian_sword": "黑曜石剑",
+   "item.easy_emerald.obsidian_bow": "黑曜石锹",
+   "item.easy_emerald.obsidian_pickaxe": "黑曜石镐",
+   "item.easy_emerald.obsidian_axe": "黑曜石斧",
+   "item.easy_emerald.obsidian_shovel": "黑曜石锹",
+   "item.easy_emerald.obsidian_hoe": "黑曜石锄",
+   "item.easy_emerald.obsidian_excavator": "黑曜石开掘铲",
+   "item.easy_emerald.obsidian_hammer": "黑曜石锤",
+   "item.easy_emerald.obsidian_paxel": "黑曜石多功能工具",
+   "item.easy_emerald.obsidian_treeaxe": "黑曜石伐木斧",
+
+   "item.easy_emerald.obsidian_head": "黑曜石头盔",
+   "item.easy_emerald.obsidian_body": "黑曜石胸甲",
+   "item.easy_emerald.obsidian_leggings": "黑曜石护腿",
+   "item.easy_emerald.obsidian_boots": "黑曜石靴子",
+
+  "item.easy_emerald.copper_sword": "铜剑",
+  "item.easy_emerald.copper_bow": "铜锹",
+  "item.easy_emerald.copper_pickaxe": "铜镐",
+  "item.easy_emerald.copper_axe": "铜斧",
+  "item.easy_emerald.copper_shovel": "铜锹",
+  "item.easy_emerald.copper_hoe": "铜锄",
+  "item.easy_emerald.copper_shears": "铜剪刀",
+  "item.easy_emerald.copper_excavator": "铜开掘铲",
+  "item.easy_emerald.copper_hammer": "铜锤",
+  "item.easy_emerald.copper_paxel": "铜多功能工具",
+  "item.easy_emerald.copper_treeaxe": "铜伐木斧",
+
+  "item.easy_emerald.copper_head": "铜头盔",
+  "item.easy_emerald.copper_body": "铜胸甲",
+  "item.easy_emerald.copper_leggings": "铜护腿",
+  "item.easy_emerald.copper_boots": "铜靴子",
+
+  "item.easy_emerald.amethyst_sword": "紫水晶剑",
+  "item.easy_emerald.amethyst_bow": "紫水晶锹",
+  "item.easy_emerald.amethyst_pickaxe": "紫水晶镐",
+  "item.easy_emerald.amethyst_axe": "紫水晶斧",
+  "item.easy_emerald.amethyst_shovel": "紫水晶锹",
+  "item.easy_emerald.amethyst_hoe": "紫水晶锄",
+  "item.easy_emerald.amethyst_excavator": "紫水晶开掘铲",
+  "item.easy_emerald.amethyst_hammer": "紫水晶锤",
+  "item.easy_emerald.amethyst_paxel": "紫水晶多功能工具",
+  "item.easy_emerald.amethyst_treeaxe": "紫水晶伐木斧",
+
+  "item.easy_emerald.amethyst_head": "紫水晶头盔",
+  "item.easy_emerald.amethyst_body": "紫水晶胸甲",
+  "item.easy_emerald.amethyst_leggings": "紫水晶护腿",
+  "item.easy_emerald.amethyst_boots": "紫水晶靴子",
+
+   "item.easy_emerald.emerald_potato": "绿宝石生命马铃薯",
+   "item.easy_emerald.ruby_potato": "红宝石战斗马铃薯",
+   "item.easy_emerald.amethyst_potato": "紫水晶幸运马铃薯",
+
+   "item.easy_emerald.emerald_juju": "绿宝石护符",
+   "item.easy_emerald.emerald_juju.line1": "提供水下呼吸效果",
+
+   "item.easy_emerald.ruby_juju": "红宝石护符",
+   "item.easy_emerald.ruby_juju.line1": "提供防火效果",
```

```suggestion
   "item.easy_emerald.ruby_juju.line1": "提供火焰保护",
```
按照字面意思，Fire Protection是指附魔https://minecraft.fandom.com/zh/wiki/%E7%81%AB%E7%84%B0%E4%BF%9D%E6%8A%A4，Fire Resistance是指状态效果 https://minecraft.fandom.com/wiki/Fire_Resistance